Automation/PublicIP/create_ip_address.ps1

```diff
@@ -0,0 +1,32 @@
+## Command to create a public ip ##
+
+# Take parameter used to create Public IP Address
+param(
+        [string] [Parameter(Mandatory=$true)] $tenantId, # Tenant ID found in the Key Vault     
+        [string] [Parameter(Mandatory=$true)] $applicationId, # Application ID found in the Key Vault  
+        [string] [Parameter(Mandatory=$true)] $secret, # Secret Key found in the Key Vault            
+        [string] [Parameter(Mandatory=$true)] $subscriptionId, # Subscription ID found in the Key Vault  
+        [string] [Parameter(Mandatory=$true)] $rgname, # Name of the resource group
```

Comment:
I think Toddy said to use longer, descriptive variable names, so he'd prefer resourceGroupName. We should check if he wants camel case or all the first letters capitalized, but low priority